### PR TITLE
Feat/corporate react cookie message #6168 (WIP)

### PR DESCRIPTION
**Pull request by creido-welly**
## In this PR
Cookie message markup and styles added from corporate repo to render basic component in prep for cookie logic work in the corporate-react app

## TODO
- refactor class names
- add style variations for buttons
- refactor styles

### CODE REVIEW

File: src/components/CookieMessage/_cookie-message.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// TODO: conform styles to Wellcome Design System
```
Comment (chriddell) [inline src/components/CookieMessage/_cookie-message.scss:1]: Presume these are pretty much copy-pasted from D7, with (spacing/colour) variables inserted?

**Review by chriddell — APPROVED** [review]
Looks good - those styles could do with looking at, but happy to tackle that in a separate PR as necessary

File: src/components/CookieMessage/_cookie-message.scss (hunk 1 above)
Comment (creido-welly) [reply src/components/CookieMessage/_cookie-message.scss:1]: yes indeed. I've literally just added some blurb about that in the description :)

**Comment by creido-welly** [comment]
> Looks good - those styles could do with looking at, but happy to tackle that in a separate PR as necessary

Absolutely. I should've flagged up the fact that this is all WIP more clearly

**Pull request merged**